client/src/components/RelatedProducts/RelatedProducts.jsx

```diff
@@ -20,79 +22,85 @@ const RelatedProducts = ({ productID }) => {
-
-    useEffect(() => {
-      if(relatedProducts.length > 0) {
-        getProductDataAsync();
-        getProductStyleAsync();
-      }
-    }, [relatedProducts])
-
-    const getProductDataAsync = async () =>{
-      const tempData =[];
-      for (var relatedProductId of relatedProducts) {
-       const data = await getProductData(relatedProductId);
-       tempData.push(data);
-      }
-      setProductData(tempData);
+  useEffect(() => {
+    if(relatedProducts.length > 0) {
+      getProductDataAsync();
+      getProductStyleAsync();
     }
+  }, [relatedProducts])
 
-    const getProductData = (relatedProduct) => {
-     return axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-sfo/products/${relatedProduct}`, {headers: config})
-        .then((product) => {
-          return product.data;
-        })
-        .catch(() =>{
-          console.log(response.err, 'err at req2')
-        })
+  const getProductDataAsync = async () =>{
+    const tempData =[];
+    for (var relatedProductId of relatedProducts) {
+      const data = await getProductData(relatedProductId);
+      tempData.push(data);
     }
+    setProductData(tempData);
+  }
 
-    console.log('this is product data', productData);
-
-    const getProductStyleAsync = async () => {
-      var tempData = [];
-      for (var relatedProductStylesId of relatedProducts) {
-        const data = await getProductStyle(relatedProductStylesId);
-        tempData.push(data);
-      }
-      console.log(tempData, 'this is tempdata')
-      setProductStyle(tempData);
-    }
+  const getProductData = (relatedProduct) => {
+    return axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-sfo/products/${relatedProduct}`, {headers: config})
+      .then((product) => {
+        return product.data;
+      })
+      .catch(() =>{
+        console.log(response.err, 'err at req2')
+      })
+  }
 
-    const getProductStyle = (relatedProduct) => {
-     return axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-sfo/products/${relatedProduct}/styles`, {headers: config})
-        .then((product) => {
-          return product.data
-        })
-        .catch(() =>{
-          console.log(response.err, 'err at req3')
-        })
+  const getProductStyleAsync = async () => {
+    var tempData = [];
+    for (var relatedProductStylesId of relatedProducts) {
+      const data = await getProductStyle(relatedProductStylesId);
+      tempData.push(data);
     }
+    setProductStyle(tempData);
+  }
 
-    console.log(productStyle)
-
-    return (
-
+  const getProductStyle = (relatedProduct) => {
+    return axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-sfo/products/${relatedProduct}/styles`, {headers: config})
+      .then((product) => {
+        return product.data
+      })
+      .catch(() =>{
+        console.log(response.err, 'err at req3')
+      })
+  }
+
+  console.log('productData', productData);
+  console.log('productStyle', productStyle);
```

Comment:
can remove logs

Reply:
will remove these, they always elude me, will work on cleaning up code more in future commits